crates/common_enums/src/enums.rs

```diff
@@ -6648,6 +6648,29 @@ pub enum TransactionType {
     Payout,
 }
 
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Eq,
+    PartialEq,
+    strum::Display,
+    strum::EnumString,
+    serde::Deserialize,
+    serde::Serialize,
+    ToSchema,
+    Default,
+)]
+#[router_derive::diesel_enum(storage_type = "db_enum")]
```

Comment:
```suggestion
#[router_derive::diesel_enum(storage_type = "text")]
```

Making this db_enum would make it difficult to add new enum variant in future.